docs/source/index.rst

```diff
@@ -1,7 +1,7 @@
 Introduction
 ============
 
-**cleanlab automatically finds and fixes errors in your ML datasets.**
+`cleanlab <https://github.com/cleanlab/cleanlab>`_ **automatically finds and fixes errors in your ML datasets.**
 
 | This reduces manual work needed to fix data issues and helps train reliable ML models on partially mislabeled datasets. ``cleanlab`` has already found thousands of `label errors <https://labelerrors.com>`_ in ImageNet, MNIST, and other popular ML benchmarking datasets, so let's get started with yours!
```

Comment:
I prefer: "partially mislabeled" -> "noisy real-world"

Reply:
Is the term "noisy" too broad? as it might also mean noise in data collection/sensor, or noise in the data features and not labels?

Reply:
I think we can already start branding this in terms of broader data quality language; branding has a delayed impact on perception and is hard to change people's perception if we don't start immediately

Reply:
SGTM! done!